backend/src/controllers/UserController.ts

```diff
@@ -55,43 +58,62 @@ export async function show(req: Request, res: Response) {
  * @returns
  */
 export async function search(req: Request, res: Response) {
```

Comment:
I'm not sure what are you trying to do with Kafka here. Aren't we storing the users on PG anymore?